### PR TITLE
Update data.json

### PR DESCRIPTION
**Pull request by EjayJanis**
Information about Digital Diamonds
Image
Description

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -2,13 +2,17 @@
   "teams": [
     {
       "name": "Digital Diamonds",
+      "repo": "https://github.com/diversify/digitaldiamonds",
+      "description": "Our application allows users to add songs to a Spotify playlist via Twitter.",
+      "image":"images/teams/logo/png",
```
Comment (johanbrook) [inline data.json:7]: This looks a bit weird :) Have you added an image to the `images/teams` folder?

**Comment by johanbrook** [comment]
I think @gopatrik has a PR open [here](https://github.com/diversify/diversify.github.io/pull/7) for you :)

File: data.json (hunk 1 above)
Comment (EjayJanis) [inline data.json:7]: My other team mate just gave me that info coz' she can't push anything on her computer. =)

**Comment by EjayJanis** [comment]
That PR hasn't been uploaded on the site....and that is the reason why I added the information on my pull request =)

**Comment by johanbrook** [comment]
Okay, I'll close that down and open this one.

But please add an image and correct the `image` attribute in `data.json` :)

**Pull request closed without merging**